packages/flutter/test/material/about_test.dart

```diff
@@ -449,6 +449,25 @@ void main() {
     expect(rootObserver.dialogCount, 0);
     expect(nestedObserver.dialogCount, 1);
   });
+
+  testWidgets(
+      "AboutListTile's child should not be offset when the icon is not specified.",
+          (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const MaterialApp(
+            home: Scaffold(
+              body: AboutListTile(
+                child: Text('About'),
+              ),
+            ),
+          ),
+        );
+
+        expect(
```

Comment:
Formatting here is a little tough to parse:
```
        expect(
          find.descendant(
            of: find.byType(AboutListTile), 
            matching: find.byType(Icon),
          ),
          findsNothing,
        );
```

Reply:
My bad. I'll add the commas

Reply:
My bad. I'll add the commas